redash/query_runner/dgraph.py

```diff
@@ -0,0 +1,127 @@
+import json
+import re
+
+try:
+    import pydgraph
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_STRING, TYPE_DATE, TYPE_DATETIME, TYPE_INTEGER, TYPE_FLOAT, TYPE_BOOLEAN
+from redash.utils import json_dumps, json_loads
+
+TYPES_MAP = {
```

Comment:
This seems unused?

Reply:
I think I'll end up using it later for the column types, but you're right, it's unused for now so I scraped it